frontends/p4/validateValueSets.h

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2022 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef _P4_VALIDATEVALUESETS_H_
+#define _P4_VALIDATEVALUESETS_H_
+
+#include "frontends/p4/typeMap.h"
+#include "ir/ir.h"
+#include "lib/error.h"
+
+namespace P4 {
+
+/**
+ * Checks that match annotations only have 1 argument which is of type match_kind.
+ */
+class ValidateValueSets final : public Inspector {
+ public:
+    ValidateValueSets() {
+        setName("ValidateValueSets");
```

Comment:
This seems (mostly) pointless as the default name for the pass will be the (qualified) name for the pass `P4::ValidateValueSets`